.evergreen/run-tests.sh

```diff
@@ -41,7 +41,7 @@ if [[ -z "${CLIENT_ENCRYPTION}" ]]; then
   unset AWS_ACCESS_KEY_ID;
   unset AWS_SECRET_ACCESS_KEY;
 else
-  npm install mongodb-client-encryption@">=2.0.0-beta.0"
+  npm install mongodb-client-encryption@">=2.0.0-beta.2"
```

Comment:
This version will need to be bumped with the next mongodb-client-encryption release.